TC_TEST_EBUS_6 failed in arm machine

v1.3 `make integrationtest` fail:

```
Event Bus Testing Publish on eventbus topics throgh MQTT internal broker 
  TC_TEST_EBUS_6: change the device status to online from eventbus
  /home/pi/gopkg/src/github.com/daixiang0/kubeedge/edge/test/integration/device/device_test.go:219
May 22 16:20:27.909: INFO: Adding Mock device to edgenode !!
May 22 16:20:27.919: INFO: PUT http://127.0.0.1:12345/devices 200 OK in 2.915896ms
May 22 16:20:29.920: INFO: client.Publish Success !!
May 22 16:20:29.920: INFO: subscribed to the topic $hw/events/device/kubeedge-device-xra6alm3m3/state/update/result
May 22 16:20:29.950: FAIL: Unmarshall failed json: cannot unmarshal number 1590135629947 into Go struct field DeviceUpdates.timestamp of type int
panic: 
Your test failed.
Ginkgo panics to prevent subsequent assertions from running.
Normally Ginkgo rescues this panic so you shouldn't see it.

But, if you make an assertion in a goroutine, Ginkgo can't capture the panic.
To circumvent this, you should call

	defer GinkgoRecover()

at the top of the goroutine that caused this panic.


goroutine 88 [running]:
github.com/onsi/ginkgo.Fail(0x2ed6000, 0x8b, 0x2d2fee8, 0x1, 0x1)
	/home/pi/gopkg/pkg/mod/github.com/onsi/ginkgo@v1.12.2/ginkgo_dsl.go:267 +0x84
github.com/kubeedge/kubeedge/edge/test/integration/utils/common.Fatalf(0x874602, 0x14, 0x2d2ff24, 0x1, 0x1)
	/home/pi/gopkg/src/github.com/daixiang0/kubeedge/edge/test/integration/utils/common/log.go:40 +0xf0
github.com/kubeedge/kubeedge/edge/test/integration/device_test.SubMessageReceived(0x998c10, 0x2e8e140, 0x997268, 0x2d50180)
	/home/pi/gopkg/src/github.com/daixiang0/kubeedge/edge/test/integration/device/device_test.go:76 +0x190
github.com/eclipse/paho%2emqtt%2egolang.(*router).matchAndDispatch.func1.3(0x2d2ff8c, 0x2e8e140, 0x997268, 0x2d50180)
	/home/pi/gopkg/pkg/mod/github.com/eclipse/paho.mqtt.golang@v1.2.0/router.go:178 +0x40
github.com/eclipse/paho%2emqtt%2egolang.(*router).matchAndDispatch.func1(0x2e2c440, 0x2e7c1b0, 0x2e8e140, 0x2e2c401)
	/home/pi/gopkg/pkg/mod/github.com/eclipse/paho.mqtt.golang@v1.2.0/router.go:180 +0x2b8
created by github.com/eclipse/paho%2emqtt%2egolang.(*router).matchAndDispatch
	/home/pi/gopkg/pkg/mod/github.com/eclipse/paho.mqtt.golang@v1.2.0/router.go:143 +0x44
Running Suite: kubeedge App Deploymet Suite
FAIL! -- 15 Passed | 1 Failed | 0 Pending | 0 Skipped
Running Suite: edgecore Suite
Integration Test Final Summary Report
===============================================
Total Number of Test cases = 16
Number of Test cases PASSED = 15
Number of Test cases FAILED = 1
==================Result Summary=======================
Integration suite has failures, Please check !!
make: *** [Makefile:131: integrationtest] Error 1

```

--- a/edge/test/integration/device/device_test.go
+++ b/edge/test/integration/device/device_test.go
@@ -37,7 +37,7 @@ var DeviceState string
 
 type DeviceUpdates struct {
 	EventID     string `json:"event_id"`
-	Timestamp   int    `json:"timestamp"`
+	Timestamp   int64  `json:"timestamp"`
 	DeviceField `json:"device"`
 }
 

